### ISSUE TITLE
Error when message contains links

### ISSUE THREAD

**Issue by marksamfd** [orkestral/venom#1686]
## Description

[Description of the bug, When Issue Happens]

## Environment

- **Venom version(s):** 4.0.11
- **Browser:** [e.g. Chrome 100]
- **OS:** [Windows 11]
- **Node version:** [node 16]

## Steps to Reproduce

on message 
**with a message that has link in it**

## Log Output

no log output because no event onmessage fired or there is an error I am not sure

```

client.onMessage((message) => {
        let content = message.content
        let hasLink = content.match(/(http|ftp|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])/ig),
        zoomLink = hasLink ? hasLink.filter(e.includes("")) : false;
        console.log(message,content,hasLink,zoomLink)
        if (hasLink && zoomLink) {

            console.log(zoomLink)

            // client
            //     .sendText(message.from, 'Welcome Venom 🕷')
            //     .then((result) => {
            //         console.log('Result: ', result); //return object success
            //     })
            //     .catch((erro) => {
            //         console.error('Error when sending: ', erro); //return object error
            //     });
        }
    });
    process.on('SIGINT', function () {
        client.close();
    });
}
```

**Comment by orkestral** [comment]
Thank you for joining our community, happy. We are currently fixing this bug, our new version will be released soon.

**Comment by orkestral** [comment]
**Fix in version 4.1.0**
